packages/core/src/integrations/inboundfilters.ts

```diff
@@ -9,8 +9,12 @@ import { defineIntegration } from '../integration';
 const DEFAULT_IGNORE_ERRORS = [
   /^Script error\.?$/,
   /^Javascript error: Script error\.? on line 0$/,
-  /^ResizeObserver loop completed with undelivered notifications.$/,
-  /^Cannot redefine property: googletag$/,
+  /^ResizeObserver loop completed with undelivered notifications.$/, // The browser logs this when a ResizeObserver handler takes a bit longer. Usually this is not an actual issue though. It indicates slowness.
+  /^Cannot redefine property: googletag$/, // This is thrown when google tag manager is used in combination with an ad blocker
+  "undefined is not an object (evaluating 'a.L')", // Random error that happens but not actionable or noticeable to end-users.
+  'can\'t redefine non-configurable property "solana"', // Probably a browser extension or custom browser (Brave) throwing this error
```

Comment:
not 100% sure, this appears _pretty_ specific, but no strong feelings, also OK to keep it here :)

Reply:
These things being specific is good :D. This particular error is in 647 projects, which is way too many to be actual application code. I am pretty sure this is brave doing things.